app/models/evmbaseline.rb

```diff
@@ -1,31 +1,49 @@
+# baseline model
 class Evmbaseline < ActiveRecord::Base
   unloadable
 
-  attr_protected :id
-
-  has_many :evmbaselineIssues, dependent: :delete_all
+  # Relations
   belongs_to :author, class_name: 'User'
   belongs_to :project
+  has_many :evmbaselineIssues, dependent: :delete_all
 
+  # attribute
+  attr_protected :id
+
+  # Validate
   validates :subject, presence: true
 
+  # Minimum start date of baseline.
+  #
+  # @return [date] minimum of start date
   def minimum_start_date
     evmbaselineIssues.minimum(:start_date)
   end
 
+  # maximum due date of baseline.
+  #
+  # @return [date] maximum of due date
   def maximum_due_date
     evmbaselineIssues.maximum(:due_date)
   end
 
+  # BAC of baseline.
+  #
+  # @return [numeric] BAC of baseline
   def bac
     evmbaselineIssues.sum(:estimated_hours).round(1)
   end
 
+  acts_as_searchable columns: ["#{table_name}.subject", "#{table_name}.description"],
+                     scope: joins(:project),
+                     permission: :view_evm_baselines,
+                     date_column: :updated_on
+
   acts_as_event title: proc { |o| l(:title_evm_tab) + ' : ' + o.subject },
-                description: proc { |o| (o.created_on < o.updated_on ? l(:label_ativity_message_edit) : l(:label_ativity_message_new)) } ,
+                description: proc { |o| (o.created_on < o.updated_on ? l(:label_ativity_message_edit) : l(:label_ativity_message_new)) },
                 datetime: :updated_on,
                 type: proc { |o| 'EvmBaseine-' + (o.created_on < o.updated_on ? 'edit' : 'new') },
-                url: proc { |o| { controller: 'evmbaselines', action: 'index', project_id: o.project } }
+                url: proc { |o| { controller: 'evmbaselines', action: :show, project_id: o.project, id: o.id } }
```

Comment:
Line is too long. [112/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.